src/api.js

```diff
@@ -48,4 +48,21 @@ const getCommentsbyArticleId = (article_id) => {
     });
 };
 
-export { getArticles, getTopics, getArticlebyId, getCommentsbyArticleId };
+const getVotesCount = (article_id) => {
+  return api.get(`/articles/${article_id}`).then(({ data: { article } }) => {
+    return article.votes;
+  });
+};
```

Comment:
This is adding in an extra API call surplus to your needs - you should already be able to access `article.votes` after using `getArticleByID` which returns the whole article object including the votes 👍